__tests__/setupTests.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Setup enzyme and some helper functions.
+ * Why this file is here:
+ * From https://create-react-app.dev/docs/running-tests/#initializing-test-environment:
+ * "If your app uses a browser API that you need to mock in your tests or if you need a global setup before running your tests, add a src/setupTests.js to your project. It will be automatically executed before running your tests."
+ */
+
+import { JSDOM } from 'jsdom'
+import Enzyme from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import { toEndWith } from './testUtils'
+require('jest-sinon')
+
+expect.extend({ toEndWith })
+
+Enzyme.configure({ adapter: new Adapter() })
+
+const { window } = new JSDOM('<!doctype html><html><body></body></html>')
```

Comment:
Other than the intersection observer, is there a reason why we need a new JSDOM object?
These internals should be handled by Jest by default, right?

Reply:
thanks I've removed this